tdrs-frontend/src/index.js

```diff
@@ -11,6 +12,10 @@ import App from './App'
 import 'uswds/dist/js/uswds'
 import './index.scss'
 
+axios.defaults.xsrfCookieName = 'csrftoken'
+axios.defaults.xsrfHeaderName = 'X-CSRFToken'
+axios.defaults.withCredentials = true
```

Comment:
These are necessary to grab the CSRF for successful API use.

Reply:
I'm not sure we need these lines any more, now that we are getting csrf from `auth_check`